docs/source/redhat-opens-install.rst

```diff
@@ -146,14 +152,12 @@ On a control node, perform the following steps:
         adduser -s /sbin/nologin -d /var/lib/etcd/ etcd
         chmod 700 /var/lib/etcd/
 
-   - Mount a ramdisk at /var/lib/etcd::
-
-        sudo mount -t tmpfs -o size=512m tmpfs /var/lib/etcd
+   - Add the following line to the bottom of ``/etc/fstab``, this will mount a
+     ramdisk for etcd at startup::
```

Comment:
@alexwlchan Can you confirm that 7cb302016cdf0261bcbe703e762dde686977d737 resolves this problem?